### PR TITLE
Hack2023 group Leaders

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
Hey all,

we need group leaders for the upcoming hackathon. Last time, you volunteered, would you be able to help out this time around again? 🥺 
Similarly, it is mostly guiding people to issues, help out if there are any problems, and make sure the slides are updated by the end of the day. You can volunteer someone from the group to present 😉 

- [x] @FranBonath 
- [ ] @mribeirodantas 
- [x] @mirpedrol
- [x] @mashehu 
- [x] @jfy133 
- [x] @louperelo 
- [x] @maxulysse 
- [ ] @drpatelh 
- [x] @Emiller88 

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1655"></a>

### CODE REVIEW

**Comment by FriederikeHanssen** [comment]
@nf-core-bot fix linting pretty please

**Comment by FriederikeHanssen** [comment]
@nf-core-bot fix linting again

**Review by mashehu — APPROVED** [review]

File: markdown/events/2023/hackathon-march-2023/index.md [hunk 1]
```diff
@@ -77,26 +77,76 @@ You are free to change groups at any time during the hackathon.
...
 
 The pipelines group will focus on any pipeline-related work. This could be a DSL2 conversion of a pipeline, a new or improved feature for a pipeline, or an entirely new pipeline (please let us register a new pipeline already before the hackathon, following [our guidelines](https://nf-co.re/docs/contributing/adding_pipelines#before-you-start)). These are typically led by the main pipeline developers if they also attend the hackathon.
 
-#### Modules
+[<i class="fab fa-slack"></i> `#hackathon-march-2023-pipelines`](https://nfcore.slack.com/archives/C04UK9FMKN0)
+
+#### Group leaders
+
+- James Fellows Yates,
+- Friederike Hanssen
```
Comment (SusiJo) [inline markdown/events/2023/hackathon-march-2023/index.md:123]:
```suggestion
- Friederike Hanssen
- Susanne Jodoin
```

File: markdown/events/2023/hackathon-march-2023/index.md [hunk 2]
```diff
@@ -77,26 +77,77 @@ You are free to change groups at any time during the hackathon.
...
+### Documentation
 
 The documentation group will focus on all things documentation related, from pipelines themselves to modules and general documentation to aid in getting started with nf-core and Nextflow. This is a great group to start if you are not sure where to begin.
 
-#### Infrastructure
+[<i class="fab fa-slack"></i> `#hackathon-march-2023-documentation`](https://nfcore.slack.com/archives/C04UGQXUK2N)
+
+#### Group leaders
+
+- Abhinav Sharma,
```
Comment (FriederikeHanssen) [inline markdown/events/2023/hackathon-march-2023/index.md:99]:
```suggestion
```

File: markdown/events/2023/hackathon-march-2023/index.md [hunk 3]
```diff
@@ -77,26 +77,76 @@ You are free to change groups at any time during the hackathon.
...
 
 The documentation group will focus on all things documentation related, from pipelines themselves to modules and general documentation to aid in getting started with nf-core and Nextflow. This is a great group to start if you are not sure where to begin.
 
-#### Infrastructure
+[<i class="fab fa-slack"></i> `#hackathon-march-2023-documentation`](https://nfcore.slack.com/archives/C04UGQXUK2N)
+
+#### Group leaders
+
+- Marcel Ribeiro-Dantas,
+- Franziska Bonath (Tuesday, Wednesday)
```
Comment (FriederikeHanssen) [inline markdown/events/2023/hackathon-march-2023/index.md:100]:
```suggestion
- Louisa Perelo (Monday)
```

File: markdown/events/2023/hackathon-march-2023/index.md [hunk 4]
```diff
@@ -77,26 +77,76 @@ You are free to change groups at any time during the hackathon.
...
 
 The documentation group will focus on all things documentation related, from pipelines themselves to modules and general documentation to aid in getting started with nf-core and Nextflow. This is a great group to start if you are not sure where to begin.
 
-#### Infrastructure
+[<i class="fab fa-slack"></i> `#hackathon-march-2023-documentation`](https://nfcore.slack.com/archives/C04UGQXUK2N)
+
+#### Group leaders
+
+- Marcel Ribeiro-Dantas,
+- Louisa Perelo (Monday)
```
Comment (FriederikeHanssen) [inline markdown/events/2023/hackathon-march-2023/index.md:100]:
```suggestion
- Franziska Bonath (Tuesday, Wednesday)
- Louisa Perelo (Monday)
```

**Pull request merged**